static/js/search_suggestion.js

```diff
@@ -238,11 +150,33 @@ function get_group_suggestions(all_people, operators) {
         return {description: description, search_string: search_string};
     });
 
+    if (!match_criteria(operators, [{operator: "is", operand: "private"}])) {
+        suggestions.push({description: "Private messages", search_string:"is:private"});
+    }
     return suggestions;
 }
 
-function get_person_suggestions(all_people, query, autocomplete_operator) {
-    if (query === '') {
+function get_person_suggestions(all_people, last, operators, autocomplete_operator) {
```

Comment:
I was kind of skeptical of the approach of combining these things into one function, but then again, if they were split, they would be around 80% repeated code. But I'll try to make this more clear in one way or another.